File: beginner/pagina02/1044/1044.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner in = new Scanner(System.in);

        int a = in.nextInt();
        int b = in.nextInt();

        if(a < b){
            int temp = b;
            b = a;
            a = temp;
        }

        if(a % b == 0){
            System.out.println("Sao Multiplos");
        } else {
            System.out.println("Nao sao Multiplos");
        }

        in.close();
    }
}

File: beginner/pagina02/1044/1044.cpp
#include <iostream>

using namespace std;

int main(){

    int a, b;
    cin >> a >> b;

    if(a < b){
        int temp = b;
        b = a;
        a = temp;
    }

    a % b == 0 ? cout << "Sao Multiplos" << endl : cout << "Nao sao Multiplos" << endl;
    
    return 0;
}